### DISCUSSION TITLE
Control the execution order of pytest

### DISCUSSION THREAD

**Discussion by ffbh123456** [pytest-dev/pytest#8243] [Q&A] [answered]
I have a lot of pytest test cases. I have labeled these cases with different labels A and B through @pytest.mark. I want them to execute all the cases labeled A first, and then execute the cases labeled B, What should I do?

**Comment by RonnyPfannschmidt** [comment, marked as answer]
The pytest_collection_modifyitems hook can be used to reorder,

You have to mark yours try last and then sort the items as you want

**Comment by ffbh123456** [comment]
Thank you, it works
